shared/notifications/index.js

```diff
@@ -0,0 +1,51 @@
+const notificationDates = {
+  1: new Date(2021, 2, 17),
```

Comment:
Why switch from ISO-8601 formatted date strings to stringified Date objects? Now the `date` value is littered with irrelevant time and time zone information, taken from the user's current browser time zone.

I'm guessing this was done because the `format` function used takes a `Date` object? Instead of doing this, I'd suggest switching back to the way it was before with string YYYY-MM-DD dates, and constructing the `Date` object just prior to the `format` invocation instead. Then our `date` state is kept simple and consistent for all users.